docs/information/binding.md

```diff
@@ -36,3 +36,11 @@ This device does not support binding (limitation of the device). A workaround is
 4. Retrieve the group from the message, which is `0xeb12` in the above example.
 5. Convert this hexadecimal number to decimal using [Hexadecimal to Decimal Converter](https://www.binaryhexconverter.com/hex-to-decimal-converter). E.g. `0xeb12` = `60178`.
 6. Add this group to `configuration.yaml` and add your device (e.g.) bulb to this group. ([documentation](./groups.md))
```

Comment:
The way you describe is the preferred one, could you remove the old method also?

Reply:
did that. can you check again?